src/components/PrimaryPanes/Outline.css

```diff
@@ -6,9 +10,11 @@
 
 .outline-list__element {
   color: blue;
+  cursor: pointer;
```

Comment:
I think we are removing `cursor: pointer` at the moment, so we can take out this line (reference issue: https://github.com/devtools-html/debugger.html/issues/3669)